NewareNDA/NewareNDA.py

```diff
@@ -27,15 +29,30 @@ def read(file, software_cycle_number=True, cycle_mode='chg'):
             'chg': (Default) Sets new cycles with a Charge step following a Discharge.
             'dchg': Sets new cycles with a Discharge step following a Charge.
             'auto': Identifies the first non-rest state as the incremental state.
+        log_level (str): Sets the modules logging level. Default: 'INFO'
+            Options: 'CRITICAL', 'ERROR', 'WARNING', 'INFO', 'DEBUG', 'NOTSET'
     Returns:
         df (pd.DataFrame): DataFrame containing all records in the file
     """
+    
+    
+    # Set up logging
+    log_level = log_level.upper()
+    if log_level in logging._nameToLevel.keys():
+        logger.setLevel(log_level)
+    else:
+        warnings.warn(f"Logging level '{log_level}' not supported; Defaulting to 'INFO'. "
+                      "Supported options are: {', '.join(logging._nameToLevel.keys())}")

```

Comment:
Should be:
```
"Supported options are: {"+', '.join(logging._nameToLevel.keys())+"}")
```
here, currently it prints: `NewareNDA.py:44: UserWarning: Logging level 'FOO' not supported; Defaulting to 'INFO'. Supported options are: {', '.join(logging._nameToLevel.keys())}`